lib/modem_key_mgmt/modem_key_mgmt.c

```diff
@@ -128,52 +128,73 @@ int modem_key_mgmt_write(nrf_sec_tag_t sec_tag,
 	}
 
 	err = write_at_cmd_with_cme_enabled(scratch_buf, NULL, 0, &state);
-
 	return translate_error(err, state);
 }
 
-int modem_key_mgmt_read(nrf_sec_tag_t sec_tag,
-			enum modem_key_mgnt_cred_type cred_type, void *buf,
-			u16_t *len)
+/* Read the given credential into the static buffer */
+static int key_fetch(nrf_sec_tag_t tag,
+		     enum modem_key_mgnt_cred_type cred_type)
 {
 	int err;
 	int written;
```

Comment:
You've made it `ret` in the write function, I'd be consistent here.

Reply:
I had renamed it to `ret` but went back on my steps.
Should be `written` everywhere now.